earth_enterprise/src/server/wsgi/serve/serve_utils.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+	#!/usr/bin/python
```

Comment:
This may not work to specify the interpreter on some platforms with the extra space at the beginning of the line.